osu.Game.Rulesets.Taiko/Difficulty/TaikoDifficultyAttributes.cs

```diff
@@ -35,10 +35,16 @@ public class TaikoDifficultyAttributes : DifficultyAttributes
         public double ColourDifficulty { get; set; }
 
         /// <summary>
-        /// The difficulty corresponding to the hardest parts of the map.
+        /// Determines the amount of difficult strain peaks present within each skill.
```

Comment:
This comment is only attached to `StaminaTopStrains` so feels kind of weird if someone is reading these comments from another file (i.e the difficulty calculator). Probably worth fixing.

Reply:
I'll just remove it, considering std doesn't have any comments for it either